.buildkite/ftr_configs.yml

```diff
@@ -289,4 +289,4 @@ enabled:
   - x-pack/performance/journeys/web_logs_dashboard.ts
   - x-pack/performance/journeys/data_stress_test_lens.ts
   - x-pack/performance/journeys/ecommerce_dashboard_saved_search_only.ts
-  - x-pack/performance/journeys/ecommerce_dashboard_tsvb_gauge_only.ts
```

Comment:
@thomasneirynck what is the reason to remove `ecommerce_dashboard_tsvb_gauge_only` from the list?

Reply:
sorry @dmlemeshko , this was a mistake when merging. https://github.com/elastic/kibana/pull/147324/commits/5b7a9efd697d418d290602734a22daa522928b98

Reply:
fixed here https://github.com/elastic/kibana/pull/147324/commits/0241d399091fc890f9e90d4c52561a2c1fc6bdba